underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -154,7 +155,13 @@ public OutputStream createDirect(String path, CreateOptions options) throws IOEx
       try {
         LOG.debug("Creating HDFS file at {} with perm {}", path, perm.toString());
         // TODO(chaomin): support creating HDFS files with specified block size and replication.
-        return FileSystem.create(mFileSystem, new Path(path),
+        Path hdfsPath = new Path(path);
+        if (!options.getCreateParent()) {
```

Comment:
Will the `FileSystem.create` call automatically create parent directories? Ideally we want to avoid the extra exists call